starport/templates/module/create/oracle/x/{{moduleName}}/client/cli/query_oracle.go.plush

```diff
@@ -0,0 +1,64 @@
+package cli
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+	"<%= modulePath %>/x/<%= moduleName %>/types"
+)
+
+// CmdOracleResult queries request by reqID
+func CmdOracleResult() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  "oracle-result",
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			id, err := strconv.ParseInt(args[0], 10, 64)
+			if err != nil {
+				return err
+			}
+			queryClient := types.NewQueryClient(clientCtx)
+			r, err := queryClient.OracleResult(context.Background(), &types.QueryOracleRequest{RequestId: id})
+			if err != nil {
+				return err
+			}
+
+			return clientCtx.PrintProto(r)
+		},
+	}
+
+	flags.AddQueryFlagsToCmd(cmd)
+
+	return cmd
+}
+
+// CmdLastRequest queries latest request
```

Comment:
we save only the last oracle id and the result data by id, but I don't it know if make sense to create a `list-oracles-result` method because we need only the last data